app/code/Magento/Backup/Model/Db.php

```diff
@@ -43,19 +45,35 @@ class Db implements \Magento\Framework\Backup\Db\BackupDbInterface
      */
     private $helper;
 
+    /**
+     * @var GetListTables
+     */
+    private $getListTables;
+
+    /**
+     * @var GetViewsBackup
+     */
+    private $getViewsBackup;
+
     /**
      * @param \Magento\Backup\Model\ResourceModel\Db $resourceDb
      * @param \Magento\Framework\App\ResourceConnection $resource
+     * @param GetListTables|null $getListTables
+     * @param GetViewsBackup|null $getViewsBackup
      * @param Helper|null $helper
      */
     public function __construct(
         \Magento\Backup\Model\ResourceModel\Db $resourceDb,
         \Magento\Framework\App\ResourceConnection $resource,
+        GetListTables $getListTables = null,
```

Comment:
Should be added to end of the construct Backward Comptibility ([Adding a constructor parameter](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/#adding-a-constructor-parameter))